test/performance/profiling.md

```diff
@@ -0,0 +1,82 @@
+# Profiling Knative Serving
+
+Knative Serving allows for collecting runtime profiling data expected
+by the pprof visualization tool. Profiling data is available for the autoscaler,
+activator, controller, webhook and for the queue-proxy
+container which is injected into the user application pod. When enabled
+Knative serves profiling data on the default port 8008 through a web server.
+
+## Steps to get profiling data:
+
+* Edit the `config-observability` ConfigMap and add `profiling.enable = "true"`:
```

Comment:
Formatting looks a little weird with the two lists here, I'd propose the following:

https://gist.github.com/markusthoemmes/7eadda432f72324aa41a6e81a898956c

(This gist also contains a reformatting of the markdown and the other suggestions I made, if you want to straight up copy it. To properly format your markdown run `prettier --write --prose-wrap=always test/performance/profiling.md`)